src/dense/generic_dense.jl

```diff
@@ -324,9 +324,16 @@ By default, simpledense
   nothing
 end
 
+"""
+ode_interpolant and ode_interpolant! dispatch
+"""
+function ode_interpolant(Θ,dt,y₀,y₁,k,cache,idxs,T::Type{Val{TI}}) where TI
+  _ode_interpolant(Θ,dt,y₀,y₁,k,cache,idxs,T)
+end
+
 function ode_interpolant(Θ,dt,y₀,y₁,k,cache::OrdinaryDiffEqMutableCache,idxs,T::Type{Val{TI}}) where TI
   if typeof(idxs) <: Number || typeof(y₀) <: Number
```

Comment:
I think this should have `typeof(y₀) <: Union{Number,SArray}`

Reply:
This is just copied, nothing new here. Can it actually happen that this is called with static vectors and a mutable cache?

Reply:
Oh I missed the mutable cache part. I don't think it's possible. I'm pretty sure any method would error, so I guess just `if typeof(idxs) <: Number` is all that's necessary.